scilpy/tractograms/tractogram_operations.py

```diff
@@ -843,3 +854,246 @@ def split_sft_randomly_per_cluster(orig_sft, chunk_sizes, seed, thresholds):
     'concatenate': 'concatenate',
     'lazy_concatenate': 'lazy_concatenate'
 }
+
+
+def _compute_difference_for_voxel(chunk_indices,
+                                  skip_streamlines_distance=False):
+    """
+    Compute the difference between two sets of streamlines for a given voxel.
+    This function uses global variable to avoid duplicating the data for each
+    chunk of voxels.
+
+    Use the function tractogram_pairwise_comparison() as an entry point.
+
+    Parameters
+    ----------
+    chunk_indices: list
+        List of indices of the voxel to process.
+    skip_streamlines_distance: bool
+        If true, skip the computation of the distance between streamlines.
+
+    Returns
+    -------
+    results: list
+        List of the computed differences in the same order as the input voxel.
+    """
+    global sft_1, sft_2, matched_points_1, matched_points_2, tree_1, tree_2, \
+        sh_data_1, sh_data_2
```

Comment:
@arnaudbore How do you want to process? Should we describe every global variable?

Reply:
We keep it as it is.